packages/fe-fpm-writer/templates/section/1.86/manifest.json

```diff
@@ -8,7 +8,11 @@
                             "content": {
                                 "body": {
                                     "sections": {
-                                        "<%- name %>": {
+                                        <% if (typeof fragmentName !== 'undefined') { %>
+                                            "<%- fragmentName %>": {
+                                        <% } else { %>
+                                            "<%- name %>": {
+                                        <% } %>
                                             "template": "<%- ns %>.<%- name %>",
```

Comment:
I suppose it should be `  "template": "<%- ns %>.<%- fragmentFile ?? name  %>",`,
but part with `"<%- name %>": {` should stay as before